### DISCUSSION TITLE
Input audio as multiple tracks and push stream only with a specific audio track

### DISCUSSION THREAD

**Discussion by tehacx** [OvenMediaLabs/OvenMediaEngine#1119] [Help]
My current setup is to stream via OBS to OvenMediaEngine RTMP and pushing this stream via RTMP to youtube. 
I'm looking for a solution to send multiple audio tracks to OvenMediaEngine, e.g. to have music in a separate track, and only push the non-music track to youtube. Similar how twitch is doing it with the VOD track feature. 
Unfortunately, it seems like OvenMediaEngine doesn't support receiving multiple audio tracks via RTMP like twitch does.
Are there any options with OvenMediaEngine that could support my desired use case?

**Comment by getroot** [comment]
OME currently does not support such a feature. What I'm wondering is, what are the use cases for not sending straight from OBS to Youtube, but with OME in the middle?

**Comment by tehacx** [comment]
Ok, thanks for the confirmation. The primary use case for my OME setup is low latency streaming with WebRTC. I'm using the RTMP push to Youtube for "free" recording.

**Comment by getroot** [comment]
Oh, I see. OME will be updated to receive multiple audio tracks from the SRT encoder. OBS also supports SRT of course so you can use this later.
